kiwmi/desktop/output.c

```diff
@@ -376,8 +388,143 @@ new_output_notify(struct wl_listener *listener, void *data)
     wl_signal_emit(&desktop->events.new_output, output);
 }
 
+void
+output_layout_change_notify(struct wl_listener *listener, void *UNUSED(data))
+{
+    struct kiwmi_desktop *desktop =
+        wl_container_of(listener, desktop, output_layout_change);
+    struct wlr_output_configuration_v1 *configuration =
+        wlr_output_configuration_v1_create();
+
+    struct kiwmi_output *output;
+    wl_list_for_each (output, &desktop->outputs, link) {
```

Comment:
Self-note: should fix what’s iterated in my rebase (also if i rebase the scene PR onto this); then also the `area!=NULL` check below makes sense.